resources/views/support/components/image-gallery.blade.php

```diff
@@ -1,62 +1,81 @@
-                                    @endphp
-                                    <div class="col-md-4 mb-3">
-                                        <div class="card gallery-card">
-                                            @if(!empty($image_asset))
-                                                <img src="{{ $image_asset['data']['full_url'] }}" alt="{{ !empty($image_asset['data']['description']) ? $block_image['data']['description'] : '' }}" load="lazy">
-                                            @else
-                                                <img
-                                                    src="{{ env('MISSING_IMAGE_URL') }}"
-                                                    load="lazy" alt="">
-                                            @endif
-                                        </div>
-                                    </div>
-                                @endforeach
-                            </div>
-                        </div>
-                    @endforeach
+            function getImageData($image_source, $image_id) {
+                if(!empty($image_id)) {
+                    foreach($image_source as $image_block) {
+                        if(!empty($image_block['directus_files_id'])) {
+                            $image_block['asset_id'] = $image_block['directus_files_id'];
+                        }
+                        if($image_block['asset_id']['id'] == $image_id) {
+                            $image_asset = $image_block['asset_id'];
+                        }
+                    }
+                }
+                return $image_asset;
+            }
+        @endphp
+        <div class="image-gallery__wrap">
+            <section class="collection-carousel splide p-0">
+                <div class="splide__arrows splide__arrows--ltr">
+                    <button class="splide__arrow splide__arrow--prev carousel-control-prev" type="button" aria-label="Previous slide" aria-controls="splide01-track">
+                        <svg height="48" viewBox="0 0 48 48" width="64" xmlns="http://www.w3.org/2000/svg">
+                            <path fill="black" d="M0 0h48v48h-48z"></path>
+                            <path fill="white" d="M14.83 30.83l9.17-9.17 9.17 9.17 2.83-2.83-12-12-12 12z"></path>
+                        </svg>
+                        <span class="visually-hidden">Previous</span>
+                    </button>
+                    <button class="splide__arrow splide__arrow--next carousel-control-next" type="button" aria-label="Next slide" aria-controls="splide01-track">
+                        <svg height="48" viewBox="0 0 48 48" width="64" xmlns="http://www.w3.org/2000/svg">
+                            <path fill="black" d="M0 0h48v48h-48z"></path>
+                            <path fill="white" d="M14.83 30.83l9.17-9.17 9.17 9.17 2.83-2.83-12-12-12 12z"></path>
+                        </svg>
+                        <span class="visually-hidden">Next</span>
```

Comment:
This visually hidden span is not necessary as you already have an `aria-label` defined on the button.

Please remove the `visually-hidden` span on the previous and next buttons.

For reference see the example on the Amplify website:

https://amplify.studio24.net/amplify/advanced-components/carousel.html